app/src/main/res/layout/activity_library.xml

```diff
@@ -11,6 +11,19 @@
     android:layout_height="match_parent"
     android:background="?attr/listBackground">
 
+
+    <TextView
+      android:id="@+id/network_text"
+      android:layout_width="wrap_content"
+      android:layout_height="match_parent"
+      android:layout_centerInParent="true"
+      android:layout_gravity="center_vertical"
+      android:gravity="center_vertical"
+      android:orientation="vertical"
+      android:paddingBottom="75dp"
```

Comment:
Hardcoded dimensions. Please extract them :)

Reply:
Have you committed correctly ? It is still hardcoded.

Reply:
oh, actually only the previous commit got pushed earlier. I hope its ok now.